### DISCUSSION TITLE
[Help Wanted] Dedicated Devtools

### DISCUSSION THREAD

**Discussion by dai-shi** [pmndrs/jotai#1637] [General] [answered]
For now, we do have jotai/devtools based on Redux DevTools (Chrome Extension).
But, the functionality is suboptimal, because the atom model and store model are different.

It would be nice to create a Jotai version of devtools.
What's in mind is pure JS (= not Chrome Extension) tools like https://tanstack.com/query/v4/docs/devtools.

Would anyone be interested in leading such a project?

**Comment by arjunvegda** [comment]
That sounds interesting! Did you have any specific features/requirements in mind?

I quickly sketched this low-fidelity UI. What do you think?

![image](https://user-images.githubusercontent.com/14841132/210192058-eaf4c003-a05a-4042-8f2a-ace78faab8bc.png)

**Reply by dai-shi** [reply]
That's great! No, I don't have specific requirements at the moment.
I can certainly give some feedback at some point.
Would you send me a DM in Discord? https://discord.gg/MrQdmzd

**Comment by vasucp1207** [comment]
@arjunvegda, great idea I am also interested in working on this project.

**Reply by arjunvegda** [reply]
Sorry for the delayed response! It'd be awesome to have you on the project 🙌. I'll DM you on Discord

**Comment by dai-shi** [comment, marked as answer]
@arjunvegda started a new project: https://github.com/jotai-labs/jotai-devtools
We can continue using this thread for discussion and also contribute there. Thanks!
